src/components/form/FormFieldSliderNumber.vue

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+/**
+ * FormFieldSliderNumber Component
+ *
+ * @description * Use this component to render an input widget for a number
+ * value with slider and manual input.
+ *
+ * Used in `RegisterChallengePayment`, `FormFieldDonation`.
+ *
+ * @props
+ * - `modelValue` (number, required): The number value.
+ *   It should be of type `number`.
+ *
+ * @events
+ * - `update:modelValue`: Emitted as a part of v-model structure.
+ *
+ * @example
+ * <form-field-slider-number v-model="number">
+ *
+ * @see [Figma Design](https://www.figma.com/design/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?node-id=6417-28987&t=gB7ERmDZorpD4TdE-1)
+ */
+
+// libraries
+import { computed, defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'FormFieldSliderNumber',
+  emits: ['update:modelValue'],
+  props: {
+    modelValue: {
+      type: Number,
+      required: true,
+      default: 390,
+    },
+    min: {
+      type: Number,
+      required: true,
+      default: 390,
```

Comment:
`default` key value should be parse from global config var `entryFeePaymentMax`.

Reply:
Updated from global config.